docs/errors.rst

```diff
@@ -0,0 +1,54 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Error Tracking
+===============
+
+Airflow can be set up to send errors to `Sentry <https://docs.sentry.io/>`__.
+
+Setup
+------
+
+First you must install sentry requirement:
+
+.. code-block:: bash
+
+   pip install 'apache-airflow[sentry]'
+
+Add your ``SENTRY_DSN`` to your environment variables. Its template resembles the following: ``'{PROTOCOL}://{PUBLIC_KEY}@{HOST}/{PROJECT_ID}'``
```

Comment:
I think it might be preferable to add this as a parameter in `airflow.cfg` instead of relying on environment variables for configuring airflow. I know this might be a bit of a refactor but I think it would provide a much more consistent experience for the users.

Reply:
Or if it is very common to set SENTRY_DSN then supporting both from the config and from _this_ environment variable might be the way to go.

Reply:
Yes; that sounds best actually. 👍

From the sentry docs [\[1\]](https://docs.sentry.io/error-reporting/configuration/?platform=python#dsn):
> If this value is not provided, the SDK will try to read it from the SENTRY_DSN environment variable.

Reply:
I fixed it so it checks the configuration file first and then defaults to the environment variable. I also updated the documentation to reflect this.